source/cloud/nvidia/brev.md

```diff
@@ -4,41 +4,66 @@ review_priority: "p0"
 
 # NVIDIA Brev
 
-The [NVIDIA Brev](https://brev.dev/) platform provides you with GPU instances on cloud providers including [Amazon Web Services](https://aws.amazon.com/) and [Google Cloud](https://cloud.google.com) with CUDA, Python, Jupyter Lab, all set up.
+The [NVIDIA Brev](https://brev.dev/) platform provides you a one stop menu of available GPU instances across many cloud providers, including [Amazon Web Services](https://aws.amazon.com/) and [Google Cloud](https://cloud.google.com), with CUDA, Python, Jupyter Lab, all set up.
 
-You can get up and running with RAPIDS in a few quicks thanks to the Brev RAPIDS quickstart.
+## Brev Instance Setup 
+There are two ways that you can get up and running with RAPIDS in a few quicks thanks to the Brev RAPIDS quickstart:
+1. Brev GPU Instances - quickly get the GPU, across most clouds, to get your work done.
+2. Brev Launchables - quickly create one-click starting, reusable instances that you customized to your MLOps needs.
 
-## Setting up your instance
+### 1. Setting up your Brev GPU Instance
```

Comment:
I don't like adding numbers to these "GPU Instance" and "Launchables" headings. It feels like step one and step two, but they are alternative options. You wouldn't do both.

Reply:
ah.  May I do Option 1 and Option 2?  I thought just bullets would be confusing.